extensions/trace-propagators/src/main/java/io/opentelemetry/extension/trace/propagation/B3Propagator.java

```diff
@@ -81,41 +84,27 @@ private B3Propagator(B3PropagatorInjector b3PropagatorInjector) {
   }
 
   /**
-   * Returns a new {@link B3Propagator.Builder} instance for configuring injection option for {@link
-   * B3Propagator}.
+   * Returns an instance of the {@link B3Propagator} that injects multi headers format.
+   *
+   * <p>This instance extracts both formats, in the order: single header, multi header.
+   *
+   * @return an instance of the {@link B3Propagator} that injects multi headers format.
    */
-  public static B3Propagator.Builder builder() {
-    return new Builder();
+  public static B3Propagator getMultiHeadersInstance() {
```

Comment:
Is `injectingMultiHeaders` a usable name? I think we need `inject` in the name or its very easy to miss that both versions behave the same on extraction.

Reply:
Sounds reasonable to me.